src/apps/shared/ordering.py

```diff
@@ -0,0 +1,39 @@
+from sqlalchemy import asc, desc
+
+__all__ = ["Ordering"]
+
+from sqlalchemy.orm import InstrumentedAttribute
+
+
+class Ordering(object):
+    actions = {
+        "+": asc,
+        "-": desc,
+    }
+
+    def __init__(self):
+        self.fields = dict()
+        for key, val in self.__class__.__dict__.items():
+            if isinstance(val, InstrumentedAttribute):
+                self.fields[key] = val
```

Comment:
The inheriting from the object is deprecated in Python 3.

Also I think it is a good idea to add the overall description for the Ordering class